content/en/tracing/profiling/_index.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Profiling
+kind: Documentation
+further_reading:
+    - link: 'https://www.datadoghq.com/blog/introducing-datadog-profiling/'
+      tags: 'Blog'
+      text: 'Introducing always-on production profiling in Datadog.'
+---
+
+<div class="alert alert-info">
+Datadog Profiling is in beta. Reach out to <a href="/help/">Datadog Support</a> if you encounter any issues or have feedback to share.
+</div>
+
+Profiling libraries are shipped within the following tracing language library. Refer to the dedicated language page to learn how to enable profile connections for your application:
+
+{{< partial name="profiling/profiling-languages.html" >}}
+
+## Profile explorer
```

Comment:
Are we actually calling this "explorer"? @nick-adams you had mentioned we were trying to phase that out at some point. Is that still true?

Reply:
happy to get the new wording implemented :)

Reply:
Ill check on this, it's just something I've heard but need to confirm. TBD

Reply:
Little e "explorer" seems to be fine when referring to stuff like this where you're searching and filtering. "Go to the Profile explorer."

But it sounds like we just want to avoid putting that explicitly as the nav item and in the title on the page, so `Profiles` not `Profile Explorer`.